scripts/ci/docker-compose/base-ports.yml

```diff
@@ -16,3 +16,9 @@
 # under the License.
 ---
 version: "3.7"
+services:
```

Comment:
This is how I implemented ports @Bowrna - I had to separate "ports" to a separate compose file and not use this compose file when running tests.

Reply:
I could see you are separating out the port, but how actually unique port values will be set for  `SSH_PORT, WEBSERVER_HOST_PORT, FLOWER_HOST_PORT` etc in case of parallel tests

Reply:
Not needed any more :). Tests are run without Port forwarding.

Reply:
Oh, how did that issue got resolved @potiuk ? Can you explain me? I couldn't understand how it was fixed by checking the code.

Reply:
it's in:

```
    def get_backend_compose_files(self, backend: str) -> list[Path]:
        backend_docker_compose_file = DOCKER_COMPOSE_DIR / f"backend-{backend}.yml"
        if backend == 'sqlite' or not self.forward_ports:
            return [backend_docker_compose_file]
        return [backend_docker_compose_file, DOCKER_COMPOSE_DIR / f"backend-{backend}-port.yml"]
```

When I run tests, ShellParams has "forward_ports" set to False - this way when set of docker-compose files are determined to be used, the `*-port.yml` files are not used.